.github/workflows/tests_without_binary.yml

```diff
@@ -0,0 +1,58 @@
+name: Testing without binary
```

Comment:
Very similar to the `tests.yml` workflow but with the c++ tests removed and a few tweaks to test only for without binaries.